crates/cli/tests/cli_run.rs

```diff
@@ -104,9 +104,11 @@ mod cli_run {
             stdin,
         );
 
-        // If there is any stderr, it should be reporting the runtime and that's it!
+        // If there is any stderr, it should only be reporting the runtime
+        // or logging the "rebuilding platform" waiting message.
         if !(compile_out.stderr.is_empty()
-            || compile_out.stderr.starts_with("runtime: ") && compile_out.stderr.ends_with("ms\n"))
+            || compile_out.stderr.starts_with("runtime: ") && compile_out.stderr.ends_with("ms\n")
+            || compile_out.stderr.contains("🔨 Rebuilding platform..."))
```

Comment:
I think this new condition is equivalent to `|| true` because the stderr string will now always contain that text. 😄 

I think what we want here is to replace the current `if` with something like:

```rust
// Strip the "Rebuilding platform..." stderr text, since we want to ignore that.
let stderr = compile_out.stderr.replacen("🔨 Rebuilding platform...", "", 1);

if !(stderr.is_empty()
    || (stderr.starts_with("runtime: ") && stderr.ends_with("ms\n")))
```

Reply:
@rtfeldman How does this look? https://github.com/roc-lang/roc/pull/3972/commits/2bf5a52146e80f33117a74599da3963252ae82bf

Reply:
This seems to work, so I'll resolve this comment and its twin.